addons/account/tests/test_account_move_reconcile.py

```diff
@@ -3847,3 +3868,160 @@ def test_caba_tax_group(self):
+            {'account_id': self.cash_basis_base_account.id,     'debit': 0.0,  'credit': 25.0, 'amount_currency': -50.0,  'currency_id': currency_id},
+            {'account_id': self.cash_basis_transfer_account.id, 'debit': 8.34, 'credit': 0.0,  'amount_currency': 16.67,  'currency_id': currency_id},
+            {'account_id': self.tax_account_1.id,               'debit': 0.0,  'credit': 8.34, 'amount_currency': -16.67, 'currency_id': currency_id},
+        ])
+
+        self.assertRecordValues(invoice.tax_cash_basis_created_move_ids.filtered(lambda x: x.date == payment_date_2).line_ids, [
+            # pylint: disable=bad-whitespace
+            {'account_id': self.cash_basis_base_account.id,     'debit': 250,   'credit': 0.0,   'amount_currency': 50.0,   'currency_id': currency_id},
+            {'account_id': self.cash_basis_base_account.id,     'debit': 0.0,   'credit': 250,   'amount_currency': -50.0,  'currency_id': currency_id},
+            {'account_id': self.cash_basis_transfer_account.id, 'debit': 83.35, 'credit': 0.0,   'amount_currency': 16.67,  'currency_id': currency_id},
+            {'account_id': self.tax_account_1.id,               'debit': 0.0,   'credit': 83.35, 'amount_currency': -16.67, 'currency_id': currency_id},
+        ])
+
+        # Check the CABA adjustment made in the receivable account's full reconcile's exchange move
+        self.assertRecordValues(
+            # pylint: disable=bad-whitespace
+            invoice.line_ids.filtered(lambda x: x.account_id.user_type_id.type == 'receivable').full_reconcile_id.exchange_move_id.line_ids,
+            [
+                {'account_id': self.cash_basis_base_account.id,     'debit': 0.05,  'credit': 0.0,  'amount_currency': 0.01,  'currency_id': currency_id},
+                {'account_id': self.cash_basis_base_account.id,     'debit': 0.0,   'credit': 0.05, 'amount_currency': -0.01, 'currency_id': currency_id},
+                {'account_id': self.tax_account_1.id,               'debit': 0.05,  'credit': 0.0,  'amount_currency': 0.01,  'currency_id': currency_id},
+                {'account_id': self.cash_basis_transfer_account.id, 'debit': 0.0,   'credit': 0.05, 'amount_currency': -0.01, 'currency_id': currency_id},
+            ]
+        )
+
+        self.assertTrue(
+            invoice.line_ids.filtered(lambda x: x.account_id == self.cash_basis_transfer_account).full_reconcile_id,
+            "The cash basis transition account line of the invoice should be fully reconciled with the CABA moves and the adjustment."
+        )
+
+        self.assertAmountsGroupByAccount([
+            # pylint: disable=bad-whitespace
+            # Account                               Balance  Amount Currency
+            (self.cash_basis_transfer_account,      0.0,     0.0),
+            (self.tax_account_1,                    -91.64,  -33.33),
```

Comment:
Which rate is this? Where does the number come from?

Reply:
It's just an addition of the amounts above: 83.35 + 8.34 - 0.05